### PR TITLE
Created Calculator UI

### PR DESCRIPTION
**Pull request by Ridwanullahi-code**
Create a directory called components.
Created Button and row components to make it resuable.
Inside components, create a new Calculator.js file
In Calculator.js, create a React component that matches this design wireframe

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,17 @@
-import logo from './logo.svg';
+/* eslint-disable */
```
Comment (cynthiainga) [inline src/App.js:1]: - [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
```
Comment (cynthiainga) [inline src/components/Button.js:1]: - [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```
Comment (cynthiainga) [inline src/components/Calculator.js:1]: - [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉

File: src/components/Rows.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
```
Comment (cynthiainga) [inline src/components/Rows.js:1]: - [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉

File: src/index.js [hunk 5]
```diff
@@ -1,18 +1,11 @@
 /* eslint-disable */
```
Comment (cynthiainga) [inline src/index.js:1]: - [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Ridwanullahi-code, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly update your `README` file by adding necessary information 🙃 😉
- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**